src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -0,0 +1,12 @@
+package com.google.ehub.data;
+
+public class CommentData {
+  private final String message;
+  private final Long timestamp;
+
+  public CommentData(String message, long timestamp){
+    this.message = message;
+    this.timestamp = timestamp;
+  }
```

Comment:
How are you supposed to access the member variables ```message``` and ```timestamp``` if they are private and the class doesn't provide any sort of public getter?

Reply:
Will any other classes need access to them?

Reply:
If you have a CommentData object and need to convert it into an Entity then you might need them, but if you are not using that right now then I guess it is fine to leave as it is.